pkg/codegen/nodejs/utilities.ts

```diff
@@ -70,7 +70,6 @@ export async function callAsync<T>(
     tok: string,
     props: pulumi.Inputs,
     res?: pulumi.Resource,
-    opts?: {property?: string},
 ): Promise<T> {
```

Comment:
I think this is right but figure it doesn't hurt to sense check -- this is backwards compatible, right @Frassle? I can't imagine a scenario where someone other than _this_ generated SDK could consume this.